linux_docker_resources/Dockerfile-RHEL

```diff
@@ -133,6 +134,7 @@ RUN dnf install \
     python3-importlib-metadata \
     $(if test ${EL_RELEASE} = 8; then echo python3-importlib-resources; fi) \
     python3-lark-parser \
+    $(if test ${EL_RELEASE} != 8; then echo python3-lttng; fi) \
```

Comment:
I think `lttng-ust` should work, but I'm not completely sure about `python3-lttng`.

Reply:
RHEL 8 has `python3-lttng`, but the problem is that it's too old and tracetools breaks if it's present. I removed it in #657. Until tracetools handles the old version gracefully (which I'd still very much like to see), this is the best workaround.